app/components/handle-input/component.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+
+export default Ember.TextField.extend({
+  _dasherizedValue: null,
+  value: Ember.computed(function(key, value){
```

Comment:
cool — I wonder if it might be helpful to add some of the [handle validations](https://github.com/aptible/api.aptible.com/blob/master/app/models/handle.rb#L4) from the API here in addition to turning " " to "-".